### PR TITLE
CLEANUP: move preparing test namespaces to a separate method

### PR DESCRIPTION
**Pull request by mm4tt**
/assign @tosi3k

### CODE REVIEW

File: clusterloader2/pkg/test/simple_test_executor.go [hunk 1]
```diff
@@ -95,14 +98,14 @@ func (ste *simpleExecutor) ExecuteTest(ctx Context, conf *api.Config) *errors.Er
...
 }
 
-// ExecuteTestSteps executes all test steps provided in configuration
-func (ste *simpleExecutor) ExecuteTestSteps(ctx Context, conf *api.Config) *errors.ErrorList {
+// prepareTestNamespaces prepares k8s namespaces for the test.
+func (ste *simpleExecutor) prepareTestNamespaces(ctx Context, conf *api.Config) error {
 	automanagedNamespacesList, staleNamespaces, err := ctx.GetClusterFramework().ListAutomanagedNamespaces()
 	if err != nil {
-		return errors.NewErrorList(fmt.Errorf("automanaged namespaces listing failed: %v", err))
+		return fmt.Errorf("automanaged namespaces listing failed: %v", err)
```
Comment (tosi3k) [inline clusterloader2/pkg/test/simple_test_executor.go:105]: Nit: s/%v/%w.

File: clusterloader2/pkg/test/simple_test_executor.go [hunk 2]
```diff
@@ -111,12 +114,15 @@ func (ste *simpleExecutor) ExecuteTestSteps(ctx Context, conf *api.Config) *erro
 			klog.Errorf("stale automanaged namespaces cleanup error: %v", errList.String())
 		}
 	}
-
 	err = ctx.GetClusterFramework().CreateAutomanagedNamespaces(int(conf.Namespace.Number))
 	if err != nil {
-		return errors.NewErrorList(fmt.Errorf("automanaged namespaces creation failed: %v", err))
+		return fmt.Errorf("automanaged namespaces creation failed: %v", err)
```
Comment (tosi3k) [inline clusterloader2/pkg/test/simple_test_executor.go:119]: Ditto about format string.

**Review by tosi3k — CHANGES_REQUESTED** [review]

File: clusterloader2/pkg/test/simple_test_executor.go (hunk 1 above)
Comment (wojtek-t) [reply clusterloader2/pkg/test/simple_test_executor.go]: seems like a fixit thing to change is in many different places :)

Comment (mm4tt) [reply clusterloader2/pkg/test/simple_test_executor.go]: Good catch, done!

File: clusterloader2/pkg/test/simple_test_executor.go (hunk 2 above)
Comment (mm4tt) [reply clusterloader2/pkg/test/simple_test_executor.go]: Done

**Pull request merged**